django/contrib/admin/options.py

```diff
@@ -1678,6 +1678,21 @@ def changelist_view(self, request, extra_context=None):
             'admin/change_list.html'
         ], context)
 
+    def get_deleted_objects(self, objs, opts, request, using):
+        """
+        Find all objects related to ``objs`` that should also be deleted.
+        ``objs`` must be a homogeneous iterable of objects (e.g. a QuerySet).
+
+        Return a nested list of strings suitable for display in the
+        template with the ``unordered_list`` filter.
+
+        By default, return django.contrib.admin.utils.get_deleted_objects.
+        This method exists to allow subclasses to override the objects that
```

Comment:
Following the existing docstring pattern of wording like "Hook for..." seems useful.